src/main/java/lotto/common/message/ErrorMessage.java

```diff
@@ -0,0 +1,36 @@
+package lotto.common.message;
+
+import lotto.common.constants.Constants;
+
+public enum ErrorMessage {
+    NOT_A_NUMBER("숫자만 입력해주세요."),
+    NOT_AN_INTEGER("정수로 변환할 수 없습니다."),
+    INPUT_REQUIRED("값을 반드시 입력해야 합니다."),
+    INVALID_INPUT_FORMAT("입력 형식이 올바르지 않습니다."),
+
+    AMOUNT_NOT_POSITIVE("금액은 양수여야 합니다."),
+    AMOUNT_NOT_MULTIPLE_OF_LOTTO_PRICE("금액이 " + Constants.LOTTO_PRICE + "의 배수가 아닙니다."),
+
+    INVALID_NUMBER_COUNT("로또 번호는 " + Constants.LOTTO_NUMBER_COUNT + "개여야 합니다."),
+    INVALID_NUMBER_RANGE("번호가 " + Constants.LOTTO_NUMBER_MIN + "이상 "
+            + Constants.LOTTO_NUMBER_MAX + "이하여야 합니다."),
```

Comment:
좀 더 가독성 좋게 아래처럼 String.format을 쓰는 것도 좋을 것 같아요!

```suggestion
    INVALID_NUMBER_RANGE(String.format("번호가 %d 이상 %d 이하여야 합니다.",
            Constants.LOTTO_NUMBER_MIN, Constants.LOTTO_NUMBER_MAX)),
```

Reply:
오 좋은 의견 감사합니다. 훨씬 가독성이 좋네요